kyuubi-hive-beeline/src/test/java/org/apache/hive/beeline/TestJSONOutputFormat.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.hive.beeline;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.ArgumentMatchers.anyInt;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.when;
+
+import java.sql.ResultSet;
+import java.sql.ResultSetMetaData;
+import java.sql.SQLException;
+import java.sql.Types;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Mockito;
+import org.mockito.invocation.InvocationOnMock;
+import org.mockito.stubbing.Answer;
+
+public class TestJSONOutputFormat {
+  private final String[][] mockRowData = {
+    {"aaa", "1", "3.14", "true", "", "SGVsbG8sIFdvcmxkIQ"},
+    {"bbb", "2", "2.718", "false", "Null", "RWFzdGVyCgllZ2cu"}
+  };
+
+  public ResultSet mockResultSet;
+  public TestBufferedRows.MockRow mockRow;
+
+  @Test
+  public final void testPrint() throws SQLException {
+    BeeLine mockBeeline = spy(BeeLine.class);
+    ArgumentCaptor<String> captureOutput = ArgumentCaptor.forClass(String.class);
+    Mockito.doNothing().when(mockBeeline).output(captureOutput.capture());
+    // when(mockBeeline.getOpts().getNumberFormat()).thenReturn("default");
+    // when(mockBeeline.getOpts().getConvertBinaryArrayToString()).thenReturn(true);
```

Comment:
why comment?

Reply:
Thank you for reviewing the code! This is also the case in the original Hive code. I will remove this comment.

Reply:
https://github.com/apache/hive/pull/1169/files LINE-49

Reply:
alright, leave it is fine then